src/js/handlers/handleCalculator.js

```diff
@@ -0,0 +1,66 @@
+import {
+  INITIAL_VALUE,
+  ERROR_MESSAGES,
+  OPERATORS,
+} from "../utils/constants.js";
+import { TOTAL } from "../utils/constants.js";
+import { $ } from "../utils/dom.js";
+import { operation } from "./operation.js";
+
+
+let numberCount = 0;
+
+export const handleCalculator = ({ target }) => {
+  switch (target.classList[0]) {
+    case "digit":
+      handleDigit(target.textContent);
+      return;
+    case "operation":
+      handleOperation(target.textContent);
+      return;
+    case "modifier":
+      $(TOTAL).textContent = INITIAL_VALUE;
+      numberCount = 0;
+      return;
+  }
+};
+
+const handleDigit = ($digit) => {
+  if (numberCount >= 3) {
+    alert(ERROR_MESSAGES.INVALID_DIGIT_LENGTH);
+    throw new Error(ERROR_MESSAGES.INVALID_DIGIT_LENGTH);
+  }
+
+  if ($(TOTAL).textContent === INITIAL_VALUE) {
+    $(TOTAL).textContent = $digit;
+    numberCount++;
+  } else {
+    $(TOTAL).textContent += $digit;
+    numberCount++;
+  }
```

Comment:
여기서 총 3번의 querySelector가 실행 됩니다. 변수로 저장해서 관리해주면 1번만 실행하게 만들 수 있답니다!
```suggestion
  const $total = $(TOTAL);
  if ($total.textContent === INITIAL_VALUE) {
    $total.textContent = $digit;
    numberCount++;
  } else {
    $total.textContent += $digit;
    numberCount++;
  }
```

Reply:
그리고 $digit은 string인데 굳이 `$`를 붙여야 하는 이유가 있을까요? 보통 `$`는 dom과 관련된 기호로 많이 사용되거든요!

Reply:
```suggestion
  $(TOTAL).textContent = Number(`${$(TOTAL).textContent}${$digit}`);
  numberCount++;
```
이렇게 표현해줄 수 있지 않을까요?

Reply:
> 여기서 총 3번의 querySelector가 실행 됩니다. 변수로 저장해서 관리해주면 1번만 실행하게 만들 수 있답니다!

와 👍👍 그렇네요 DOM요소 조작 메서드는 최소한으로 사용하는 게 좋겠어요!

Reply:
> 그리고 $digit은 string인데 굳이 `$`를 붙여야 하는 이유가 있을까요? 보통 `$`는 dom과 관련된 기호로 많이 사용되거든요!

이부분은 제가 $의 사용을 잘못 이해하고 있었던 것 같아요...클라이언트 쪽에서 오는 건 전부 $ 를 붙이는 줄 알았어요. Document(HTML)의 정보만 $를 붙이는 걸로 이해하면 될까요??

Reply:
> 

마지막 코드를 쓰면 한 줄로 예쁘게 줄일 수 있는 것 같아요! 그런데 연산자를 추가한 후엔 Number로 전환이 안되어 NaN이 발생하게 됩니다 ㅜㅜ 이부분을 개선해보려고 했으나 fail....